docs/source/1.0/spec/aws/aws-json.rst.template

```diff
@@ -192,6 +192,77 @@ In ``awsJson1_1``, servers SHOULD only send the error's
 for both protocols. See `Operation error serialization`_ for full details on how to
 deserialize errors for |quoted shape name|.
 
+.. smithy-trait:: aws.protocols#awsQueryError
+.. _aws.protocols#awsQueryError-trait:
+
+-------------------------------------
+``aws.protocols#awsQueryCompatible`` trait
+-------------------------------------
+
+Summary
+    Enables backward compatibility for :ref:`aws.protocols#awsQueryError <aws.protocols#awsQueryError-trait>`
```

Comment:
Let's get more descriptive here. How does it enable backward compat? When would an AWS team use this trait?